.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/AssignmentInCondition:
+  Description: Don't use assignment in conditions.
+  StyleGuide: "#safe-assignment-in-condition"
+  Enabled: true
+  AllowSafeAssignment: true
+
```

Comment:
I actually prefer the split assignment and conditional checks. I think you underestimate how much I despise assignment in conditionals and how much of my life has been lost to such constructs. But unless there's a cop to prevent that, I guess I'll have to live with it.